packages/financial-templates-lib/src/logger/SlackTransport.ts

```diff
@@ -174,7 +176,7 @@ class SlackHook extends Transport {
     this.axiosInstance = axios.create({ proxy: opts.proxy });
   }
 
-  async log(info: any, callback: () => void) {
+  async log(info: any, callback: (error?: unknown) => void): Promise<void> {
```

Comment:
`info: any` is not ideal but I guess this would require bigger changes that we don't necessarily want to do right?

Reply:
Yes, any is used in winston transport base class. And each transport might have its own types and we don't control all of transport implementations (e.g. GCP logger). Best we can do is add type guards in our transport implementations - that's what I used in the newest DiscordTicketTransport